perfkitbenchmarker/providers/openstack/os_disk.py

```diff
@@ -81,6 +88,38 @@ def WaitForVolumeCreation(resource, volume_id):
     raise errors.Resource.RetryableCreationError(msg)
 
 
+disk.RegisterDiskTypeMap(providers.OPENSTACK, DISK_TYPE)
+
+
+class OpenStackDiskSpec(disk.BaseDiskSpec):
+  """Object holding the information needed to create an OpenStack disk.
+
+  Attributes:
+    disk_size: None or int. Size of the disk in GB.
+  """
+
+  CLOUD = providers.OPENSTACK
+
+  @classmethod
+  def _ApplyFlags(cls, config_values, flag_values):
+    """Modifies config options based on runtime flag values.
+
+    Can be overridden by derived classes to add support for specific flags.
+
+    Args:
+      config_values: dict mapping config option names to provided values. May
+          be modified by this function.
+      flag_values: flags.FlagValues. Runtime flags that may override the
+          provided config values.
+    """
+    super(OpenStackDiskSpec, cls)._ApplyFlags(config_values, flag_values)
+    if flag_values['openstack_volume_size'].present \
```

Comment:
Instead of using `\` to split the line, surround the boolean expression with parenthesis `( )`.  Break the line before the `and` binary operator, add an indentation level.[1]

[1] https://www.python.org/dev/peps/pep-0008/#indentation